### DISCUSSION TITLE
How to imbed Zoom-invitation into add2cal

### DISCUSSION THREAD

**Discussion by 19john59** [add2cal/add-to-calendar-button#364] [Q&A] [answered]
Hello add2cal-Community,

I am not a coder. I'm fairly good at managing my WordPress site. I'm using add2cal and am grateful to its creator, and plan to make a donation. 

Is it possible to imbed a Zoom-invitation into add2cal, so that folks who participate in my Zoom-session need only go to their calender and click on that link?

Thanks.

John

**Comment by jekuer** [comment, marked as answer]
You can put the link into the description.
{url}yourUrl{/url} will make a clickable url out of it.
You can even specify a link text by using | as separator.

For example: description="{url}https://www.google.com|your Link Text{/url}

**Comment by 19john59** [comment]
Perfect, Jens !

Looking forward to sending
you some money, once I earn
some.


John

+49 171 860 6637
understand-culture.com
